c++/src/Reader.cc

```diff
@@ -854,6 +854,8 @@ namespace orc {
 
   class ReaderImpl : public Reader {
   private:
+    const int64_t epochOffset;
+
```

Comment:
ReaderImpl::epochOffset is not used (we seem to call getEpochOffset() everywhere).